build/build-treemap.js

```diff
@@ -18,7 +18,7 @@ function buildStrings() {
   const locales = require('../lighthouse-core/lib/i18n/locales.js');
   // TODO(esmodules): use dynamic import when build/ is esm.
   const utilCode = fs.readFileSync(LH_ROOT + '/lighthouse-treemap/app/src/util.js', 'utf-8');
-  const {UIStrings} = eval(utilCode.replace('export ', '') + '\nmodule.exports = TreemapUtil;');
+  const {UIStrings} = eval(utilCode.replace(/export /g, '') + '\nmodule.exports = TreemapUtil;');
```

Comment:
heh, I guess this technically works.

Reply:
Is there a different solution you had in mind?